src/app/odasstream/odas_stream.py

```diff
@@ -22,24 +21,21 @@ def __init__(self, parent=None):
         self.isRunning = False
         
 
-    def start(self):
-        print("Starting Odas stream...")
+    def start(self, odasPath, micConfigPath):
+        print('Starting Odas stream...')
 
-        try:  
-            
-            odasPath = AppSettings.getValue("odasPath")
-            micConfigPath = AppSettings.getValue("micConfigPath")
+        try:
 
-            if not odasPath or odasPath == "":
-                raise Exception("odasPath needs to be set in the settings")
+            if not odasPath or odasPath == '':
+                raise Exception('odasPath needs to be set in the settings')
 
-            if not micConfigPath or micConfigPath == "":
-                raise Exception("micConfigPath needs to be set in the settings")
+            if not micConfigPath or micConfigPath == '':
```

Comment:
we only need this if statement to cover both conditions:

    if not micConfigPath: